src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lotto {
+    private List<Integer> numbers;
+
+    public Lotto(LottoNumbers numbers) {
+        this.numbers = numbers.generateRandomNumbers();
+    }
+
+    public int getMatchCount(LottoNumbers numbers) {
+        return (int) numbers.getWinningNumbers().stream().filter(n -> this.numbers.contains(n)).count();
+    }
+
+    public List<Integer> getLottoNumbers() {
+        return this.numbers;
+    }
```

Comment:
```suggestion
    public List<Integer> getLottoNumbers() {
        return numbers;
    }
```
this 는 생략해도 무방할 것 같아요~